content/patterns/slider/examples/css/slider-rating.css

```diff
@@ -14,84 +14,88 @@
 }
 
 .rating-slider svg .focus-ring {
-  fill: #eee;
+  fill: currentcolor;
   stroke-width: 0;
   fill-opacity: 0;
 }
 
-.rating-slider svg .star {
+.rating-slider svg .value {
   stroke-width: 2px;
   stroke: currentcolor;
   fill-opacity: 0;
 }
 
-.rating-slider svg .fill-left,
-.rating-slider svg .fill-right {
-  stroke-width: 0;
-  fill-opacity: 0;
-}
-
-.rating-slider[aria-valuenow="5"] svg .star {
+.rating-slider svg .label {
+  font-size: 90%;
+  font-family: sans-serif;
   fill: currentcolor;
-  fill-opacity: 1;
 }
 
-.rating-slider[aria-valuenow="0.5"] svg .star-1 .fill-left {
-  fill: currentcolor;
-  fill-opacity: 1;
+.rating-slider svg .description {
+  font-size: 90%;
+  fill: canvastext;
 }
 
-.rating-slider[aria-valuenow="1"] svg .star-1 .star {
+.rating-slider svg .current .value {
   fill: currentcolor;
   fill-opacity: 1;
 }
 
-.rating-slider[aria-valuenow="1.5"] svg .star-1 .star,
-.rating-slider[aria-valuenow="1.5"] svg .star-2 .fill-left {
-  fill: currentcolor;
-  fill-opacity: 1;
+.rating-slider svg .current .label {
+  fill: white;
```

Comment:
Why is this one hardcoded to white? Isn't there a named color we could use here?